app/views/site/studentFilterSelection.html.haml

```diff
@@ -54,6 +63,11 @@
         
       
     :javascript
+      $(window).load(function(){
+      $('#cover').fadeOut(2000);
```

Comment:
will increase the tme for the FadeOut as per the requirement. That won't be
a problem. The other thing u r suggesting willl get complicated